dpgen/generator/run.py

```diff
@@ -700,11 +700,14 @@ def _make_fp_vasp_inner (modd_path,
     system_index.sort()
 
     fp_tasks = []
-    cluster_cutoff = jdata['cluster_cutoff'] if 'use_clusters' in jdata and jdata['use_clusters'] else None
+    cluster_cutoff = jdata['cluster_cutoff'] if jdata.get('use_clusters', False) else None
+    # skip save *.out if make_fp_out is False, default is True
+    make_fp_out = jdata.get("make_fp_out", True)
```

Comment:
The meaning of "make_fp_out" is not clear.... How about "detailed_report_make_fp" ?  
If the user do not want a detailed report as what we are providing now, a brief report on the percentage of candidate, accurate and failed should be given.